src/tests/ftest/util/ec_utils.py

```diff
@@ -108,12 +109,14 @@ def ec_container_create(self, oclass):
         # update object class for container create, if supplied explicitly.
         ec_object = get_data_parity_number(self.log, oclass)
         rf = "rf:{}".format(ec_object['parity'])
-        if self.container.properties.value is None:
+        if self.ec_container.properties.value is None:
             self.ec_container.properties.update(rf)
         else:
-            self.ec_container.properties.update("{},{}".format(self.container.properties.value, rf))
+            self.ec_container.properties.update("{},{}"
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string